janusgraph-solr/src/main/java/org/janusgraph/diskstorage/solr/SolrIndex.java

```diff
@@ -405,6 +412,20 @@ public void mutate(Map<String, Map<String, IndexMutation>> mutations, KeyInforma
         }
     }
 
+    private String getDualMappingName(String key) {
+        String fieldName;
+        if (key.endsWith("_t")) {
```

Comment:
combine the if else into a ternary operation and return without creating a local variable